paddle/fluid/pir/dialect/op_generator/ops_api_gen.py

```diff
@@ -142,6 +142,7 @@
     'uniform_random_batch_size_like',
     'c_reduce_min',
     'c_reduce_min_',
+    'push_sparse_v2',
```

Comment:
```suggestion
    'push_sparse_v2',
    'push_sparse_v2_',
```
标记了inplace的op会生成两个版本，带下划线的是inplace版本，所以这里要屏蔽两个

Reply:
收到